rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -824,6 +824,10 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		const QString text = var_list[0].toString();
 		const bool enabled = text == "Cubeb" || text == "XAudio2" || text == "FAudio";
 		ui->enableBuffering->setEnabled(enabled);
+		if (!enabled)
+		{
+			ui->enableBuffering->setChecked(false);
```

Comment:
Don't do this. This will actively disable the buffering option.
Which means you'll have to enable it again when changing the audio backend